modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/jdbc/JdbcResultSetSelfTest.java

```diff
@@ -0,0 +1,1105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.runner.app.jdbc;
+
+import java.io.InputStream;
+import java.io.Reader;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.sql.Blob;
+import java.sql.Clob;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.NClob;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.GregorianCalendar;
+import org.apache.ignite.internal.tostring.S;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Result set test.
+ */
+@SuppressWarnings({"FloatingPointEquality", "AssertWithSideEffects"})
+public class JdbcResultSetSelfTest extends AbstractJdbcSelfTest {
+    /** SQL query. */
+    private static final String SQL =
+        "SELECT 1::INTEGER, true, 1::TINYINT, 1::SMALLINT, 1::INTEGER, 1::BIGINT, 1.0::FLOAT, 1.0::DOUBLE, 1.0::DOUBLE, " +
+            "'1', '1', '1901-02-01'::DATE, '01:01:01'::TIME, 1::TIMESTAMP;";
+
+    /** Statement. */
+    private Statement stmt;
+
+    /**
+     * Create the connection ant statement.
+     *
+     * @throws Exception if failed.
+     */
+    @BeforeEach
+    public void beforeTest() throws Exception {
+        Connection conn = DriverManager.getConnection(URL);
+
+        stmt = conn.createStatement();
+
+        assert stmt != null;
+        assert !stmt.isClosed();
+    }
+
+    /**
+     * Close the connection and statement.
+     *
+     * @throws Exception if failed.
+     */
+    @AfterEach
+    public void afterTest() throws Exception {
+        if (stmt != null) {
+            stmt.getConnection().close();
+
+            stmt.close();
+
+            assert stmt.isClosed();
```

Comment:
Let's use assertTrue() here.

Reply:
I replaced the assert function with assertTrue and other methods on all tests.